SConscript

```diff
@@ -48,6 +48,8 @@ if env['noadaptivesync']==1:
     env.Append(CPPDEFINES    = 'NOADAPTIVESYNC')
 if env['l2_security']==1:
     env.Append(CPPDEFINES    = 'L2_SECURITY_ACTIVE')
+if env['disable_ch_hop']!=11 and env['disable_ch_hop']<=26:
```

Comment:
Could be better using `fix_channel` instead of `disable_ch_hop`? It's strange to have `disable_ch_hop` to have a value.

Also the conditions is wrong.

Reply:
Ups, Sorry that was typo that got in when re-writing. addessed.